lint-staged.config.js

```diff
@@ -1,8 +1,6 @@
 module.exports = {
-  'demo/**/*.js': ['pnpm run fix:eslint', 'pnpm run format:ts', 'git add -u'],
-  'packages/**/*.{js,ts}': [
-    'pnpm run fix:eslint',
-    'pnpm run format:ts',
+  '*.{js,ts,tsx}': [
+    'biome check --write --no-errors-on-unmatched', // Format, sort imports, lint, and apply safe fixes
```

Comment:
`biome check` is another CLI command that is recommended to use in lint-staged, it checks and you can optionally modify files without committing.

The check command is nice because it shows the idea behind biome which is to act as a single tool and not separate tools. you can format, sort imports and lint with a single command. I like it

Reply:
also `tsx` was not included so I added it 😄

Reply:
I updated this to include all files  with `*` hope that's okay. it's easier to maintain and allow for biome to evolve without manual updates